addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/core/LxWsStateUpdateEvent.java

```diff
@@ -11,18 +11,22 @@
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
 /**
  * An event received from Loxone Miniserver with control's state update
  *
  * @author Pawel Pieczul - initial contribution
  *
  */
+@NonNullByDefault
```

Comment:
Would it be less configuration, to not use this one here?

Reply:
This will result in many incompatibilities in classes that use this class and are non null by default (or we mark all method arguments in this class as non null, which is even more ugly)

Reply:
Okay I did not completely realise the impact 🙂